packages/kbn-test/src/functional_tests/cli/run_tests/args.test.js

```diff
@@ -22,6 +22,14 @@ import { createAbsolutePathSerializer } from '@kbn/dev-utils';
 
 expect.addSnapshotSerializer(createAbsolutePathSerializer(process.cwd()));
 
+const INITIAL_TEST_ES_FROM = process.env.TEST_ES_FROM;
+beforeEach(() => {
+  process.env.TEST_ES_FROM = 'snapshot';
```

Comment:
Are we concerned with any of the possible ramifications of doing this when running parallel Jest tests?

Reply:
No, jest mocks out the env so each test file has its own.